### PR TITLE
Search Bar Component and Service Provider List

### LINKED ISSUE #32: Create search view
## REQUIREMENTS / DESCRIPTION

You will create the search view where the user client can searches service provider and can see in a list the results.
You will join the search bar component, the service provider list component to create this view, and button tab navigator.

- [ ] Create a search view.
- [ ] Join the search bar component, the service provider list component to create this view, and button tab navigator.

## DESIGN

The search view should looks like this:
searchView

### PR DESCRIPTION
**Pull request by DerekAyala**
# Description
This pull request introduces two significant enhancements that enrich the user experience: the "Search Bar" component and the "Service Provider List" feature. Together, these features provide users with a streamlined and efficient way to discover and explore service providers.

## Search Bar Component:

- Implemented the "SearchBar" component, enabling Client users to perform quick and targeted searches for service providers.
- Designed a visually appealing and responsive user interface for the search bar.
- Created the foundation for integrating search functionality in future updates.

## Service Provider List Feature:

- Developed the "ServiceProviderList" component, offering users a comprehensive view of service providers.
- Designed the layout of the service provider list, ensuring an aesthetically pleasing presentation.
- Laid the groundwork for additional functionalities, such as accessing detailed service provider profiles.

## Screenshots
image

### CODE REVIEW

File: CtasApp/src/img/lic.png
Comment (Tole15) [inline CtasApp/src/img/lic.png:1]: NICE LOOKING MAN

**Review by Eduardoespl — APPROVED** [review]
LGMT! 🚀

**Review by Lizzflores — APPROVED** [review]
LGTM

**Review by Tole15 — APPROVED** [review]
LGTM 
image

**Pull request merged**